.github/workflows/release.yml

```diff
@@ -97,7 +97,7 @@ jobs:
       matrix:
         include:
           # See https://cibuildwheel.readthedocs.io/en/stable/options/#build-skip for more details.
-          - os: macos-latest
+          - os: macos-13 # macos-13 is the last non-large x86-based runner. Need to look into cross-compiling.
```

Comment:
Don't understand what `non-large` means here.

Reply:
A better term would probably be standard runners. Runners that aren't `-large`, or `-xlarge`.

You can opt into large or x-large runners like `macos-13-large`. For intel, the standard (non-large) runners stop at MacOS 13 with `macos-13-xlarge` being Arm, but standard and large being intel. MacOS 14 flips to Arm for standard, and xlarge, but large is intel.